airflow/config_templates/default_airflow.cfg

```diff
@@ -250,6 +250,13 @@ daemon_umask = 0o077
 # Example: dataset_manager_kwargs = {{"some_param": "some_value"}}
 # dataset_manager_kwargs =
 
+# (experimental) Whether components should use Airflow Internal API for DB connectivity.
```

Comment:
Why do we have this change, this seems unrelated ?

Reply:
Quite the contrary. They are very much related. They were (correctly) automatically added by pre-commit.

Those parameters have min-version = 2.7. This was implemented as part of the feature flags for AIP-44 and AIP-52, but in order to not comploicate it further  we restricted automated moving parameters from .yaml to config based on "version-added" > currrent version. We moved the related parameters that were scheduled for 2.7 to have "version-added=2.7.0" so they started to appear only when we moved the version to 2.7.0.